rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/mute/muterunner.kt

```diff
@@ -170,7 +170,7 @@ fun RpgParserResult.executeMuteAnnotations(
             }
         }
     }
-    interpreter.execute(cu, parameters)
+    interpreter.execute(cu, parameters, reinitialization = true)
```

Comment:
Related to previous review.

Reply:
It's an error... I'll fix it

Reply:
Done, solved by https://github.com/smeup/jariko/pull/165/commits/000252be92866db6d80440bff29ee33bad91c0e2